Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -390,6 +393,39 @@ void GameList::ShowContextMenu(const QPoint&)
   menu->exec(QCursor::pos());
 }
 
+void GameList::ShowHeaderContextMenu(const QPoint&)
+{
+  static const QMap<QString, bool*> columns{
```

Comment:
Since this is the same one as `MenuBar::AddListColumnsMenu`, can (or should!) we move that to a shared location instead?

Reply:
I though the same but I didn't know where to put it. I could move it to `Settings.cpp` as a private static variable and add a getter method. Both `GameList.cpp` and `MenuBar.cpp` inherit from `Settings.cpp`

My only issue is that I don't think that's the purpose of `Settings.cpp`. Do you have any suggestions of where I could move it to?